components/ILIAS/Contact/PRIVACY.md

```diff
@@ -0,0 +1,88 @@
+# Contact Service Privacy
+
+This documentation does not warrant completeness or correctness. Please report any
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+- The "Contact" component of ILIAS provides a channel for users to contact each other.
+- The component only handles the contact and shares of information via the contact functionality.
+  There are multiple other ways of context related contacts in ILIAS that are not part of this component, like course 
+  member/tutor relations or test correction processes, etc
```

Comment:
When I read your version, I had problems understanding the part "shares of information."

Can you slightly change to: 
"The component only handles the contact and sharing of information via the contact functionality. There are multiple other types of context-related contacts in ILIAS that are not part of this component, such as course member/tutor relations or test correction processes, etc."